api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddMemberRoleToPersonGenerator.java

```diff
@@ -84,7 +84,7 @@ FieldOptions getRoleActivityFieldOptions(VitroRequest vreq) throws Exception {
 			//UQAM Replacing the above hard coding assigment by a dynamic assigment that takes into account the linguistic context
 	       ConstantFieldOptions filedOptions = GeneratorUtil.buildConstantFieldOptions(vreq, DESCRIBE_QUERY);
 	       return filedOptions;
-	}
+    }
```

Comment:
Also here.. I believe this line is a part of Michel's functional update... and can likely be removed from this pull-request.